examples/gno.land/r/demo/foo721/foo721.gno

```diff
@@ -0,0 +1,122 @@
+package foo721
+
+import (
+	"std"
+
+	"gno.land/p/demo/grc/grc721"
+	"gno.land/p/demo/ufmt"
+	"gno.land/r/demo/users"
+)
+
+var (
+	admin std.Address = "g1us8428u2a5satrlxzagqqa5m6vmuze025anjlj"
+	foo               = grc721.NewBasicNFT("FooNFT", "FNFT")
+)
+
+func init() {
+	mintNNFT(admin, 10)                                     // @administrator (10)
+	mintNNFT("g1var589z07ppjsjd24ukm4uguzwdt0tw7g47cgm", 5) // @hariom (5)
```

Comment:
What do you think about leaving out test code from the example contract?

You should be able to mint an NFT using an address that has enough balance for it